### DISCUSSION TITLE
How to check which IP in a pool was used to send X emails

### DISCUSSION THREAD

**Discussion by alex-pommier** [postalserver/postal#2266] [Development discussions]
I have my server configured to use 4 IPs in a pool i've setup. Checking on google's postmaster tools I see some reputation issues on 1 IP in my pool. I'm curious to know now whether emails sent these past months all sent from 1 IP or if the IP Table rotated between the IPs i've configured the email server to use. How can I check and confirm if this is the case or not?

**Comment by traderpedroso** [comment]
> I have my server configured to use 4 IPs in a pool i've setup. Checking on google's postmaster tools I see some reputation issues on 1 IP in my pool. I'm curious to know now whether emails sent these past months all sent from 1 IP or if the IP Table rotated between the IPs i've configured the email server to use. How can I check and confirm if this is the case or not?

On the email pool you can set priority, for example the one you want to send less just set 50 and other 100!  And to check the IP your SMTP server using,  you can send email to this tool on the log you can check the IP follow the URL for the tool [email check IP URL](https://www.appmaildev.com/en/dkim)

**Reply by alex-pommier** [reply]
I've set the priority to 25 for all my servers as I'd like the emails sent through postal to rotate equally between all 4 of my IPs. I'm just concerned that postal is only sending out via 1 of my configured IPs.

**Reply by traderpedroso** [reply]
> I've set the priority to 25 for all my servers as I'd like the emails sent through postal to rotate equally between all 4 of my IPs. I'm just concerned that postal is only sending out via 1 of my configured IPs.

On my case I set all to 100 and working good rotating every single IP i think its doesn't matter if 25 or 100 priority is equal just set the one is not good less than other

**Comment by willpower232** [comment]
I don't believe this information is recorded outside of the queued_messages database table and I'm not sure how long the rows in that table hang around but probably worth having a look.
